pandas/tests/io/formats/style/test_matplotlib.py

```diff
@@ -260,6 +260,12 @@ def test_background_gradient_gmap_wrong_series(styler_blank):
         styler_blank.background_gradient(gmap=gmap, axis=None)._compute()
 
 
+def test_background_gradient_nullable_dtypes():
+    # test when gmap as converted ndarray is bad shape
```

Comment:
Could you add the Github issue number here?

Reply:
Done, used the same commenting style as the rest of this file